src/components/cards/boolean-card.tsx

```diff
@@ -103,14 +124,18 @@ export const BooleanCard = ({
     await patchPlayer(patch);
   }
 
+  let checkedClass = completed
+    ? "border-green-900 bg-green-500/20 hover:bg-green-500/30 dark:bg-green-500/10 hover:dark:bg-green-500/20"
+    : "border-neutral-200 bg-white dark:border-neutral-800 dark:bg-neutral-950 hover:bg-neutral-100 dark:hover:bg-neutral-800";
+
   return (
     <ContextMenu>
       <ContextMenuTrigger asChild>
         <button
-          className={cn(
-            "relative flex select-none items-center justify-between rounded-lg border px-5 py-4 text-neutral-950 shadow-sm hover:cursor-pointer dark:text-neutral-50",
-            checkedClass,
-          )}
+          className={
+            "relative flex select-none items-center justify-between rounded-lg border px-5 py-4 text-neutral-950 shadow-sm hover:cursor-pointer dark:text-neutral-50 " +
```

Comment:
"relative flex select-none items-center justify-between rounded-lg border px-5 py-4 text-neutral-950 shadow-sm hover:cursor-pointer dark:text-neutral-50" +
 
 another extra space between 50 and "
 
 Also should be put into a css file?